py4cast/datasets/poesy/__init__.py

```diff
@@ -692,41 +658,41 @@ def sample_list(self):
         print("Start forming samples")
         terms = list(
             np.arange(
-                self.settings.term["start"],
-                self.settings.term["end"],
-                self.settings.term["timestep"],
+                METADATA["TERMS"]["start"],
```

Comment:
It comes from METADATA ? So it means it would not be configurable by the user (METADATA is not supposed to be modified right ?)

Reply:
yes, indeed. These values describe the dataset poesy as a whole and not a user-defined, training-time configuration.

Reply:
should be ok now ?